typescript/web/src/components/website/hero.tsx

```diff
@@ -7,7 +7,7 @@ import {
   Heading,
   Stack,
   Text,
-  useColorModeValue as mode,
+  useColorModeValue,
```

Comment:
You should let it as it is for now to stay consistent with the majority of our code, or refactor every other place that make that rename

Reply:
On such invasive and "soft" relations things (_the `mode` name is local to the file importing the function and `useColorModeValue` is imported in a lot of files_), I try to be more progressive and do the changes over the time.

For this, specific case, it's not always renamed as `mode` in the existing code. For example, I've nearly always used `useColorModeValue` since my arrival and replaced `mode` as I saw it in the code I had to change.

This because:
* React hooks function names must begin with `use`. We're bypassing some verifications by not using the prefix.
* `mode` is a way too generic word IMO. If the function name length was actually a problem (_e.g. makes the code less readable_), then I'd use a more precise name such as `useColorMode` or eventually `useColor`.
* We can still use a proxy function in `utils` with a shorter name instead of forcing an implicit consistency (_using the `mode` name in each file importing `useColorModeValue`_) such as the one you've found.

But you're right that it's inconsistent for now👍 I'll do the switch to `useColorModeValue` once for all in an additional commit on this PR so that we can forget about `mode` 🚀

Reply:
Done